src/components/BorrowerProfile/LendCta.vue

```diff
@@ -500,9 +510,17 @@ export default {
 		},
 		unreservedAmount(newValue, previousValue) {
 			// set initial selected value for sub 25 loan if shown
-			if (newValue !== previousValue && previousValue === '' && newValue < 25) {
+			if (this.completeLoan && this.isBetween25And100) {
+				this.selectedOption = Number(this.unreservedAmount).toFixed();
+			} else if (newValue !== previousValue && previousValue === '' && newValue < 25) {
 				this.selectedOption = parseInt(newValue, 10);
 			}
+		},
+		isCompleteLoanActive() {
```

Comment:
This should be renamed so it doesn't share the name with an existing property.

Reply:
That is the watcher of existing computed property.

Reply:
🤦